core/src/main/java/io/github/tcdl/msb/impl/ObjectFactoryImpl.java

```diff
@@ -42,4 +50,18 @@ public Requester createRequester(String namespace, RequestOptions requestOptions
     public ResponderServer createResponderServer(String namespace,  MessageTemplate messageTemplate, ResponderServer.RequestHandler requestHandler) {
         return ResponderServerImpl.create(namespace, messageTemplate, msbContext, requestHandler);
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ChannelMonitorAggregator createChannelMonitorAggregator(Callback<AggregatorStats> aggregatorStatsHandler) {
```

Comment:
Thanks a very good catch. Timer will continue sending periodic heartbeats after msbContext shutdown but it will fail because the broker connection is broken.